src/libstd/Cargo.toml

```diff
@@ -25,6 +25,9 @@ profiler_builtins = { path = "../libprofiler_builtins", optional = true }
 unwind = { path = "../libunwind" }
 hashbrown = { version = "0.4.0", features = ['rustc-dep-of-std'] }
 
+[target.'cfg(not(all(target_arch = "wasm32", target_vendor = "unknown", target_os = "unknown")))'.dependencies]
```

Comment:
Due to the usage of `if cfg!` the `getrandom` crate needs to be avialable on wasm32-unknown-unknown, and I think that's fine in that this header shouldn't be necessary anyway

Reply:
Without an enabled `dummy`,`stdweb` or `wasm-bindgen` feature the current version of `getrandom` will emit a compilation error when compiled for `wasm32-unknown-unknown`. If I am not mistaken without this config `getrandom` will be compiled unconditionally even if it was not used in the crate.